.github/workflows/ci.yml

```
@@ -42,7 +42,7 @@ jobs:
           distribution: adopt
           cache: 'sbt'
 
-      - uses: sbt/setup-sbt@v1
+      - uses: sbt/setup-sbt@v1.1.6
```

Why was this change necessary? If I understand correctly, it should be possible to set `@v1.1` to automatically resolve to the latest patch updates.

```suggestion
      - uses: sbt/setup-sbt@v1.1
```